packages/cli-platform-android/src/commands/runAndroid/index.ts

```diff
@@ -42,6 +54,35 @@ export type AndroidProject = NonNullable<Config['project']['android']>;
 async function runAndroid(_argv: Array<string>, config: Config, args: Flags) {
   link.setPlatform('android');
 
+  let {packager, port} = args;
+
+  const packagerStatus = await isPackagerRunning(port);
+
+  if (
+    typeof packagerStatus === 'object' &&
+    packagerStatus.status === 'running'
+  ) {
+    if (packagerStatus.root === config.root) {
+      packager = false;
+      logAlreadyRunningBundler(port);
+    } else {
+      const result = await handlePortUnavailable(port, config.root, packager);
+      [port, packager] = [result.port, result.packager];
+    }
+  } else if (packagerStatus === 'unrecognized') {
+    const result = await handlePortUnavailable(port, config.root, packager);
+    [port, packager] = [result.port, result.packager];
+  }
+
+  if (packager) {
+    await startServerInNewWindow(
```

Comment:
await causes the CLI to hang indefinitely. Must be called without await.
Check existing comment at line no 105